packages/netsuite-adapter/src/config.ts

```diff
@@ -118,6 +146,18 @@ export const getConfigFromConfigChanges = (failedToFetchAllAtOnce: boolean, fail
   if (_.isEmpty(suggestions)) {
     return undefined
   }
+
+  const clientConfigSuggestion = suggestions[FETCH_ALL_TYPES_AT_ONCE] !== undefined
+    ? _.pickBy({
+      [FETCH_ALL_TYPES_AT_ONCE]: suggestions[FETCH_ALL_TYPES_AT_ONCE],
+      [SDF_CONCURRENCY_LIMIT]: currentConfig[CLIENT_CONFIG]?.[SDF_CONCURRENCY_LIMIT],
+      [MAX_ITEMS_IN_IMPORT_OBJECTS_REQUEST]: currentConfig[CLIENT_CONFIG]
+        ?.[MAX_ITEMS_IN_IMPORT_OBJECTS_REQUEST],
+      [FETCH_TYPE_TIMEOUT_IN_MINUTES]: currentConfig[CLIENT_CONFIG]
+        ?.[FETCH_TYPE_TIMEOUT_IN_MINUTES],
+    }, values.isDefined)
+    : currentConfig[CLIENT_CONFIG]
```

Comment:
nit: I'm not sure I'm following the difference between the two cases correctly - is it intended to prioritize `suggestions[FETCH_ALL_TYPES_AT_ONCE]` over `currentConfig[CLIENT_CONFIG]?.[FETCH_ALL_TYPES_AT_ONCE]`? if so - maybe use 
```suggestion
  const clientConfigSuggestion = suggestions[FETCH_ALL_TYPES_AT_ONCE] !== undefined
    ? _.pickBy({
      ...(currentConfig[CLIENT_CONFIG] ?? {}),
      [FETCH_ALL_TYPES_AT_ONCE]: suggestions[FETCH_ALL_TYPES_AT_ONCE],
    }, values.isDefined)
    : currentConfig[CLIENT_CONFIG]
```
or
```suggestion
  const clientConfigSuggestion = _.pickBy(_.merge(
      {},
      currentConfig[CLIENT_CONFIG],
      { [FETCH_ALL_TYPES_AT_ONCE]: suggestions[FETCH_ALL_TYPES_AT_ONCE] },
    ), values.isDefined)
```
to make sure if more parameters are added they won't be missed?

Reply:
Great idea!
Implemented it also for the adapter-config level.